HousingSearchApi/V1/Interfaces/SearchAssetsQueryContainerOrchestrator.cs

```diff
@@ -0,0 +1,47 @@
+using HousingSearchApi.V1.Boundary.Requests;
+using HousingSearchApi.V1.Gateways.Models;
+using HousingSearchApi.V1.Infrastructure;
+using Nest;
+using System.Collections.Generic;
+
+namespace HousingSearchApi.V1.Interfaces
+{
+    public class SearchAssetsQueryContainerOrchestrator : ISearchQueryContainerOrchestrator<GetAssetListRequest, QueryableAsset>
```

Comment:
This class probably shouldn't be in the Interfaces namespace.

Reply:
We decided to keep already iplemented arhitecture, that's why these classes are here. Because previously we use to have this files structure.  
Are we able to change project structure?

Reply:
We will probably have to if this is the case.  We can find out.  If this is not something we are introducing it can be done on a new branch.